src/main/java/seedu/address/logic/parser/EventCommandParser.java

```diff
@@ -56,5 +63,11 @@ public EventCommand parse(String args) throws ParseException {
         return new EventCommand(event);
     }
 
+    /**
+     * Returns true if the start time is before the end time.
+     */
+    private static boolean isStartTimeBeforeEndTime(LocalDateTime startTime, LocalDateTime endTime) {
```

Comment:
hmm i already implemented this in lessonCommandParser, no repeated code, abstract this method out if needed